services/QuillLMS/app/queries/snapshots/top_concepts_assigned_query.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Snapshots
+  class TopConceptsAssignedQuery < TopXQuery
+    def run_query
+      QuillBigQuery::Runner.execute(utilizing_subquery)
+    end
+
+    def utilizing_subquery
```

Comment:
I think I wrote this on Brendan's testing PR, but you are overwriting the parent method for other methods, so you should consider doing that here as well to cut down on needing new methods, e.g.
```ruby
def query
 <<-SQL
      SELECT value, SUM(count) AS count
        FROM (#{super})
        GROUP BY value
        ORDER BY count DESC
        LIMIT #{NUMBER_OF_RECORDS}
      SQL
end